src/overlay.c

```diff
@@ -1,62 +1,62 @@
 // SPDX-License-Identifier: GPL-2.0-only
 #include <assert.h>
-#include <wlr/render/pixman.h>
 #include "labwc.h"
 #include "overlay.h"
 #include "view.h"
 
 static void
-create_overlay_rect(struct seat *seat, struct overlay_rect *rect, int fill,
-		float bg_color[4], int border_width, float border_colors[3][4])
+create_overlay_rect(struct seat *seat, struct overlay_rect *rect,
+		bool bg_enabled, float bg_color[4], bool border_enabled,
+		int border_width, float border_colors[3][4])
```

Comment:
If we use a substruct in `theme.h` which defines an overlay and then just have a `region` or `edge` overlay pointer would simplify the args and the caller quite a bit. Not a requirement for me though.

Reply:
Good idea. Done.